### PR TITLE
Update get_line_length serializer method to count only un-started meetings (#270)

### PR DESCRIPTION
**Pull request by ssciolla**
The PR aims to resolve issue #270.

### CODE REVIEW

File: src/officehours_api/serializers.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ class Meta:
                   'allowed_backends']
 
     def get_line_length(self, obj):
-        return obj.meeting_set.count()
+        return len([obj for obj in obj.meeting_set.all() if obj.status != MeetingStatus.STARTED])
```
Comment (jlost) [inline src/officehours_api/serializers.py:157]: You might be able to replace the `all` with a `filter`, which would be a bit more performant (offloads work to postgres).

**Review by jlost — APPROVED** [review]
LGTM!

File: src/officehours_api/serializers.py (hunk 1 above)
Comment (ssciolla) [reply src/officehours_api/serializers.py:157]: I wanted to do that, but using a model method (`status`) makes it less straightforward. Looks like you can write a queryset manager that could do it nicely, but I'm not a huge fan of those (yet at least).

https://stackoverflow.com/questions/2276768/django-query-filtering-from-model-method

I also thought about defining a method on `Queue` that would get all the un-started meetings (which we could also use in `Meeting.line_place`), but `Meeting` and `MeetingStatus` are defined after `Queue`, so I'd need to mess with `models.py` more than I really wanted to.

**Comment by ssciolla** [comment]
Thanks, I just modified the loop variable for clarity. Merging now...

**Pull request merged**